src/utils.rs

```diff
@@ -499,4 +516,14 @@ mod test {
             ]
         );
     }
+
+    #[test]
+    fn data_vec_test() {
+        from_bigint_str![3];
```

Comment:
I don't get why is this here

Reply:
That macro is needed so that the trait `MaybeRelocatable::from(([u8; u32], integer)` is implemented

Reply:
Removed line: https://github.com/lambdaclass/cairo-rs/pull/400/commits/2cdcae7b2ef38863c744a86f090c7b4edd296ed0